leaderboard/README.md

```diff
@@ -111,7 +113,7 @@ for lang in "${langs[@]}"; do
         task=multiple-$lang
     fi
 
-    gen_suffix=generations_$task\_$model.json
+    gen_suffix=generations_$task\_$model\_$task.json
```

Comment:
> because during evaluation we call `--load-generations_path` which can be anything.

Right however, current **README steps for Evaluation** passes `$gen_suffix` variable  in `--load-generations_path` argument
https://github.com/bigcode-project/bigcode-evaluation-harness/blob/642c57f138390828c743a748469c2a82d9bff72b/leaderboard/README.md?plain=1#L114-L121

Since `$gen_suffix` is missing the `_task` suffix, Running evaluations results in the following error
 
 
<img width="889" alt="image" src="https://github.com/bigcode-project/bigcode-evaluation-harness/assets/36822111/0b374a6c-ccee-4c5d-8e9e-6bd98b65bcda">

After adding `_task` suffix in `$gen_suffix`, evaluations run successfully. 

I was able to run evaluations for the [Artigenz-Coder-DS-6.7B here](https://huggingface.co/spaces/bigcode/bigcode-models-leaderboard/discussions/66/files) after these changes

Reply:
it shouldn't throw an error if you used `save_generations_path=generations_$task\_$model.json` in the generations